### DISCUSSION TITLE
Ability to view blocked user's profile and comments

### DISCUSSION THREAD

**Discussion by DraconicNEO** [Docile-Alligator/Infinity-For-Reddit#1337] [Ideas]
Hello I think it would be a good feature to be able to view posts and comments of users who are blocked/have blocked you by pulling the profile data either from logged out reddit or from pushshift in the same way that Removed and deleted comments are retrieved.

**Comment by Wladefant** [comment]
Well, yes if we can detect whether you are blocked.

Here are three blog posts by reddit about the new blocking experience since last year:
- https://www.reddit.com/r/changelog/comments/p2ezy4/bringing_more_visibility_to_comments_from_blocked/
- https://www.reddit.com/r/blog/comments/s71g03/announcing_blocking_updates/
- https://www.reddit.com/r/redditsecurity/comments/w3u2b5/update_on_user_blocking/

**Reply by DraconicNEO** [reply]
It should be relatively easy to detect if you're blocked since the comments will show up as `[Unavailible]` instead of `[Deleted]` or `[Removed]`, shouldn't be much harder than it is to detect and fetch removed and deleted comments from pushshift.
